.github/workflows/main-release.yml

```diff
@@ -0,0 +1,69 @@
+name: Main Release
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - 'deployment/**'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up python
+        uses: actions/setup-python@v4
```

Comment:
check out if caching for poetry should be included

Reply:
Will create an issue for this, and see if we should do it in all repos.